cpp/util/etcd.cc

```diff
@@ -499,15 +487,21 @@ EtcdClient::WatchUpdate::WatchUpdate()
 }
 
 
-void EtcdClient::WatchInitialGetDone(WatchState* state, util::Status status,
-                                     const Node& node, int64_t etcd_index) {
-  WatchInitialGetAllDone(state, status, {node}, etcd_index);
+void EtcdClient::WatchInitialGetDone(WatchState* state, GetResponse* resp,
+                                     Task* task) {
+  unique_ptr<GetResponse> resp_deleter(resp);
+  GetAllResponse* const all_resp(new GetAllResponse);
+  all_resp->etcd_index = resp->etcd_index;
```

Comment:
![](https://s-media-cache-ak0.pinimg.com/originals/26/31/11/26311156159f94ddf350d6e81121f520.jpg)
An object `new`/`delete` effectively just to pass in an int, but I guess it's pretty infrequent.